### PR TITLE
P8 calvin expenses

### PR DESCRIPTION
**Pull request by 1mozolacal**
## Description
The list of all expenses. Still need to add tap functionality but bare bones is done.

## Related Issue
- [ ] This pull request relates to at least one particular issue

  
  - [issue name](https://github.com/ps-toronto-team-4/.github/issues/8)

## Checks
- [X] My pull request adheres to the code style of this project
- [ ] My code requires changes to documentation
- [ ] I have updated the documentation as required
- [X] All the tests have passed

### CODE REVIEW

**Review by HKanwal — APPROVED** [review]

File: screens/ExpensesScreen.tsx [hunk 1]
```diff
@@ -11,97 +12,69 @@ import {
...
-          paddingHorizontal: 30,
-          paddingVertical: 20,
-          backgroundColor: 'white',
-        }}
-      >
-
+      <View style={styles.expenseItemDisplayContainer}>
         <Text style={{ flex: 1, fontSize: 24 }}>
-          {title}
+          {description}
```
Comment (HKanwal) [inline screens/ExpensesScreen.tsx:77]: Display the category name instead of the description. Refer to the Figma.

**Review by HKanwal — CHANGES_REQUESTED** [review]
See requested change.

File: screens/ExpensesScreen.tsx (hunk 1 above)
Comment (1mozolacal) [reply screens/ExpensesScreen.tsx]: fixed

File: screens/ExpensesScreen.tsx [hunk 2]
```diff
@@ -11,97 +12,69 @@ import {
...
-          paddingHorizontal: 30,
-          paddingVertical: 20,
-          backgroundColor: 'white',
-        }}
-      >
-
+      <View style={styles.expenseItemDisplayContainer}>
         <Text style={{ flex: 1, fontSize: 24 }}>
-          {title}
+          {category?.name}
```
Comment (HKanwal) [inline screens/ExpensesScreen.tsx:77]: Render 'Uncategorized' if category is undefined.

**Review by HKanwal — CHANGES_REQUESTED** [review]
See requested change.

**Review by HKanwal — APPROVED** [review]

**Pull request merged**